.github/workflows/tag_image_push_rhel.yml

```diff
@@ -23,6 +23,33 @@ on:
           - ''
           - 'false'
           - 'true'
+      DRY_RUN:
+        description: 'Skip pushing the images to remote registry'
+        default: 'false'
+        type: choice
+        options:
+          - 'false'
+          - 'true'
+  workflow_call:
+    inputs:
+      HZ_VERSION:
+        type: string
+        description: 'Version of Hazelcast to build the image for, e.g. 5.1.1, 5.0.1, 4.2.3'
```

Comment:
Taking https://hazelcast.atlassian.net/browse/DI-342 into account, we would end up with unpublishing HZ_VERSION while testing PRs. I would suggest either
-  skip running `tag_image_push_rhel.yml` during PR checks
-  modify it to not push anything to RHEL in dry-run and run only `simple-smoke-test.sh` instead
-  push the image with a temp tag like `pr-check-813`, but it depends on the approach we'll choose to fix permanently DI-342

Reply:
> Taking https://hazelcast.atlassian.net/browse/DI-342 into account, we would end up with unpublishing HZ_VERSION while testing PRs. I would suggest either
> 
> * skip running `tag_image_push_rhel.yml` during PR checks
> * modify it to not push anything to RHEL in dry-run and run only `simple-smoke-test.sh` instead
> * push the image with a temp tag like `pr-check-813`, but it depends on the approach we'll choose to fix permanently DI-342

Agreed.

> * skip running `tag_image_push_rhel.yml` during PR checks


https://github.com/hazelcast/hazelcast-docker/pull/813/commits/a6e86d0c75f913f472b2915f17efd16701e3234f